tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3
+envlist = py27,py34,py35,py36,pypy,pypy3
 
 [testenv]
 commands =
-    python setup.py -q test -q
-deps = 
-    zope.testing
-    six
+    zope-testrunner --test-path=src
+deps =
+    .[test]
```

Comment:
FWIW, my concern with `envlist = py27,...,coverage` where `[coverage]` does something like `coverage combine && coverage report` is that it breaks under a default invocation of `detox` 😢 . (I haven't looked to see if there's a way in tox.ini to tell detox to run just one environment after all the others.)